.github/workflows/_build_rosetta.yaml

```diff
@@ -74,15 +74,15 @@ jobs:
           driver-opts: |
             image=moby/buildkit:v0.10.6
 
-      - name: Pull and validate BASE_IMAGE=${{ env.BASE_IMAGE }} contains BASE_LIBRARY=${{ inputs.BASE_LIBRARY }}
-        shell: bash -x -e {0}
-        run: |
-          PYTHON_PACKAGE=${{ inputs.BASE_LIBRARY }}
-          if [[ $PYTHON_PACKAGE == "pax" ]]; then
-            PYTHON_PACKAGE=paxml
-          fi
-          docker run --rm ${{ env.BASE_IMAGE }} python -c "import $PYTHON_PACKAGE"
-
+      # TODO: Can't run a cuda image yet
```

Comment:
Use `runs-on: [self-hosted, small-builder]` and it will work. I added the CUDA container runtime to our self-hosted runners 😁

Reply:
Awesome! I can uncomment. Thanks for the upgrade~

Reply:
I tried, but it didn't work. Am I missing something?

https://github.com/NVIDIA/JAX-Toolbox/actions/runs/5082514197/jobs/9132336819